tests/AbstractTestCase.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Illuminate\Tests;
+
+use Mockery;
+use Illuminate\Support\Carbon;
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractTestCase extends TestCase
+{
+    /**
+     * @var \Illuminate\Support\Carbon
+     */
+    protected $now;
+
+    protected function setUp()
+    {
+        Carbon::setTestNow($this->now = Carbon::now('UTC'));
```

Comment:
That assumes that someone is not running a forked version of PHPUnit though.

Adding it would not be any overhead, but future proof incase something ends up in there in the package in the future?

But your right - it wont matter right now.